proto/api.go

```diff
@@ -308,14 +307,139 @@ func (sr *ReverseScanResponse) Verify(req Request) error {
 	return nil
 }
 
+// GetInner returns the Request contained in the `oneof`.
+func (ru BatchRequest_RequestUnion) GetInner() Request {
```

Comment:
This is a big step backwards. If we have to write our own methods with big switch statements like this, what do we actually gain by using `oneof`? `oneof` and `onlyone` are conceptually similar, but it doesn't look like they're trying to solve the same problem.